src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -297,67 +314,46 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
-            <div v-if="profileMods.known.length === 0 || profileMods.unknown.length > 0" class="notification is-warning margin-top">
-                <p v-if="profileMods.known.length === 0">
-                    None of the packages in the profile were found on Thunderstore:
-                </p>
-                <p v-else>
-                    Some of the packages in the profile were not found on Thunderstore:
-                </p>
-
+            <div v-if="knownProfileMods.length === 0 || profileMods.unknown.length > 0" class="notification is-warning">
+                <p>These packages in the profile were not found on Thunderstore and will not be installed:</p>
                 <p class="margin-top">{{ unknownProfileModNames }}</p>
 
-                <p v-if="profileMods.known.length > 0" class="margin-top">
-                    These packages will not be installed.
-                </p>
-                <p v-else class="margin-top">
+                <p v-if="knownProfileMods.length === 0" class="margin-top">
                     Ensure the profile is intended for the currently selected game.
                 </p>
-
-                <p class="margin-top">
-                    Refreshing the mod list from Thunderstore might solve this issue.
-
-                    <div v-if="$store.getters['download/activeDownloadCount'] > 0">
-                        <span>
-                            However, the mod list can't be refreshed while the are mod downloads in progress.
-                            Please wait for the downloads to finish before continuing.
-                        </span>
-                    </div>
-
-                    <div v-else>
-                        <span v-if="$store.state.tsMods.modsLastUpdated">
-                            The mod list was last refreshed on {{ valueToReadableDate($store.state.tsMods.modsLastUpdated) }}.
-                        </span>
-
-                        <br />
-
-                        <span v-if="$store.state.tsMods.isThunderstoreModListUpdateInProgress">
-                            {{ $store.state.tsMods.thunderstoreModListUpdateStatus }}
-                        </span>
-                        <span v-else-if="$store.state.tsMods.thunderstoreModListUpdateError">
-                            Error refreshing the mod list:
-                            {{ $store.state.tsMods.thunderstoreModListUpdateError.message }}.
-                            <a @click="$store.dispatch('tsMods/syncPackageList')">Retry</a>?
-                        </span>
-                        <span v-else>
-                            Would you like to
-                            <a @click="$store.dispatch('tsMods/syncPackageList')">refresh now</a>?
-                        </span>
-                    </div>
-                </p>
             </div>
+
+            <p v-if="knownProfileMods.length > 0" class="margin-bottom">These packages will be installed:</p>
```

Comment:
Minor: this seems redundant if the yellow box is not shown, as the text is quite similar to the header right above it. I'd show this only when the yellow box is shown.